foundations/installations/setting_up_git.md

```diff
@@ -55,6 +55,10 @@ First, you'll need to install Homebrew.  Make sure you have checked the requirem
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ~~~
 
+Note: On an Apple Silicon Mac you will have an extra step to take.
```

Comment:
I actually wonder if this is even an extra step for silicon devices. I think it might be relevant for all device.

Reply:
With my limited knowledge, reading the following it seems like it's an extra step in arm64 setups; https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh

I feel like all this goes a bit over my head. But there definitely wasn't an extra step on my older iMac.

Reply:
Yea, it's actually sorta weird how they don't handle this automatically for ARM64 macOS users, since the script *does* detect for it in several places already. You are right though, the "Add Homebrew to your PATH..." message will only appear for Apple Silicon users.